### DISCUSSION TITLE
Help me pull images from docker like nginx or postgres without (/imagename)

### DISCUSSION THREAD

**Discussion by lehakatana** [goharbor/harbor#21137] [Q&A] [answered]
Hello, i pulled easily image from docker hub goharbor/harbor-core with tag dev
command for this is: docker pull goharbor/harbor-core:dev 
with this settings:
![image](https://github.com/user-attachments/assets/ae984468-f850-4670-ab5d-de7238d4fab8)

But i can't get image postgres with tag 17-alpine, for this image command is 
docker pull postgres:17-alpine
link : [https://hub.docker.com/_/postgres ](url)

seems harbor can't take image from "account name" without image name, help pls, i tried many options, nothing downloaded

**Comment by MinerYang** [comment, marked as answer]
dup of https://github.com/goharbor/harbor/issues/21139
